### DISCUSSION TITLE
ArangoDB Community Project Slack Channel?

### DISCUSSION THREAD

**Discussion by lostpebble** [cw00dw0rd/ArangoBnB#25] [Ideas]
@cw00dw0rd , I was just thinking that it could aid in quick communication about various things we're working on if we had a channel in slack to chat. Github Issues and such are good for the most part- but sometimes there are small nuances you'd like to communicate more directly / quickly.

Is it maybe possible to create something along these lines on the ArangoDB slack which we can then all join? (and whoever else wants to get involved).

**Comment by cw00dw0rd** [comment]
I would like to keep as much discussion on GitHub as possible, to keep the conversations in one place, but I will see about adding a channel on Slack as well. Feel free to message me directly `Chris.ArangoDB` on the community channel.

**Reply by cw00dw0rd** [reply]
I have created a `community-projects` channel in the community slack.
@lostpebble and @couds would you ping me on slack. We would like to send out a swag box as a thank you for helping out with the project.

**Reply by GetOutExploreMore** [reply]
Hello Chris where are you at in Slack? I tried ArangoBnB and ArangoDB and was not able to find the ArangoBnB channel.

Tim Schmidt

**Reply by cw00dw0rd** [reply]
Hi Tim,

Thanks for reaching out!

We created the [community-projects](https://arangodb-community.slack.com/archives/C01MLH491UM) channel for discussion related to these projects, so there isn't a specific one for ArangoBnB.
My name in the community slack is Chris.ArangoDB as well.
